hedera-node/src/main/java/com/hedera/services/state/StateAccessor.java

```diff
@@ -41,7 +43,7 @@
 import java.time.Instant;
 
 public class StateAccessor {
-	private StateChildren children = new StateChildren();
+	private volatile StateChildren children = new MutableStateChildren();
 
 	public StateAccessor() {
```

Comment:
could remove `public StateAccessor(final ServicesState initialState)` constructor as it only used in unit test

Reply:
Done.